### PR TITLE
KEYCLOAK-18440 Improve logging for token introspection

### PR DESCRIPTION
**Pull request by y-tabata**
JIRA ticket: https://issues.redhat.com/browse/KEYCLOAK-18440

### CODE REVIEW

**Review by hmlnarik — DISMISSED** [review]
Thank you @y-tabata!

File: services/src/main/java/org/keycloak/protocol/oidc/AccessTokenIntrospectionProvider.java [hunk 1]
```diff
@@ -81,6 +84,7 @@ protected AccessToken verifyAccessToken(String token) throws OAuthErrorException
 
             accessToken = verifier.verify().getToken();
         } catch (VerificationException e) {
+            logger.debug("JWT check failed: " + e.getMessage());
```
Comment (hmlnarik) [inline services/src/main/java/org/keycloak/protocol/oidc/AccessTokenIntrospectionProvider.java:87]: Prefer
```suggestion
            logger.debugf("JWT check failed: %s", e.getMessage());
```

**Comment by y-tabata** [comment]
@hmlnarik thanks for the review!
I rebased and changed as you mentioned.

**Review by hmlnarik — APPROVED** [review]
Thanks @y-tabata !

**Pull request merged**